fem/bilininteg_dgtrace_pa.cpp

```diff
@@ -43,9 +43,10 @@ static void PADGTraceSetup2D(const int Q1D,
    auto W = w.Read();
    auto qd = Reshape(op.Write(), Q1D, 2, 2, NF);
 
-   MFEM_FORALL(f, NF, // can be optimized with Q1D thread for NF blocks
+   MFEM_FORALL(tid, Q1D*NF,
```

Comment:
This may be a bit more clear:
```suggestion
   MFEM_FORALL_3D(f, NF, Q1D, 1, 1,
```